client/my-sites/promote-post/components/post-item/style.scss

```diff
@@ -1,46 +1,96 @@
+$post-item-background-color: var( --color-surface );
```

Comment:
Used the posts page as a starting point, few adpatations yo match our needs and then ran it through chromes coverage tool to remove any unecessary rules